appserver/pom.xml

```diff
@@ -692,6 +692,16 @@
                 <artifactId>helidon-microprofile-config</artifactId>
                 <version>${helidon-config.version}</version>
             </dependency>
+            <dependency>
+            <groupId>io.helidon.common</groupId>
```

Comment:
As this will be updated anyway to correct indentation - can't we import `helidon-bom` instead?